data-reconciliation-app/Makefile

```diff
@@ -10,20 +10,29 @@ build: ## 🔨 Build the Application
 	@echo -e "\e[34m$@\e[0m" || true
 	@npm run build
 
-build-virtual: build ## 📦 Build Virtual container image from Dockerfile
+run-env-setup: create-jwt-configs build  ## 🔨 Create the initialization configs and Build the Application
```

Comment:
Is this no longer needed? could the other targets just `create-jwt-configs build`  instead?

Reply:
Done